print-engine/src/main/java/de/tillhub/printengine/pax/PaxPrinterController.kt

```diff
@@ -142,6 +142,7 @@ class PaxPrinterController(
      * Start printer and prints data in buffer.This is synchronous interface.
      */
     override fun start() {
+        printerState.value = PrinterState.Busy
         printerService.start()
         printerService.init()
```

Comment:
Maybe add one more between start and init `printerState.value = getPrinterState()` or maybe doesn't have much sense since we are adding at the end. Im curious whats in between state though